### PR TITLE
Update attributes.rst

### PR DESCRIPTION
**Pull request by kylebrenner**

### CODE REVIEW

File: reference/conanfile/attributes.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Attributes
 name
 ----
 This is a string, with a minimum of 2 and a maximum of 50 characters (though shorter names are recommended), that defines the package name. It will be the ``<PkgName>/version@user/channel`` of the package reference.
-It should match the following regex ``^[a-zA-Z0-9_][a-zA-Z0-9_\+\.-]$``, so start with alphanumeric or underscore, then alphanumeric, underscore, +, ., - characters.
+It should match the following regex ``^[a-zA-Z0-9_][a-zA-Z0-9_\+\.-]*$``, so start with alphanumeric or underscore, then alphanumeric, underscore, +, ., - characters.
```
Comment (danimtb) [inline reference/conanfile/attributes.rst]: Actually, the real regex is this one: https://github.com/conan-io/conan/blob/master/conans/model/ref.py#L23
```suggestion
It should match the following regex ``^[a-zA-Z0-9_][a-zA-Z0-9_\+\.-]{1,50}$``, so start with alphanumeric or underscore, then alphanumeric, underscore, +, ., - characters.
```

**Comment by danimtb** [comment]
Hi @kylebrenner and thanks for the PR.

Please check the suggestion above so we can merge this contribution asap 😄

**Comment by kylebrenner** [comment]
I believe this should be good now, but it's my first pr so let me know if you need anything else.

**Comment by danimtb** [comment]
thanks for the fast response. You have to hit the "Commit suggestion" button (will add a new commit with the change suggested) and then "resolve conversation".

That should be enough 😄

**Comment by kylebrenner** [comment]
done, thanks

**Comment by danimtb** [comment]
Now it is perfect. Thanks a lot!

**Pull request merged**